builder/Dockerfile

```diff
@@ -32,19 +32,15 @@ RUN apk --no-cache add alpine-sdk \
                        libgcrypt-dev \
                        mariadb-dev \
                        cmake \
+                       gcc \
```

Comment:
Strictly speaking, I don't think we _need_ `gcc` here, as it's part of both `build-base` and `alpine-sdk`.